icecap/mk/realm.mk

```diff
@@ -39,6 +39,39 @@ define realm_crate_body
 			--out-dir $(bin_dir)
  endef
 
-.PHONY: inner-runtime-manager
-inner-runtime-manager:
+.PHONY: runtime-manager
+runtime-manager: sysroot-install
 	$(call realm_crate_body,runtime-manager)
+
+.PHONY: sysroot-install
+sysroot-install: sysroot
```

Comment:
Fingerprints of these rlibs change, so repeated invocations of this target will result in an undesirable accumulation of stale rlibs. Which will rustc choose for building and linking against?

Reply:
Let's fix this in a separate PR.

Reply:
👍